i18n/language/pluralspec_test.go

```diff
@@ -512,6 +516,52 @@ func TestUkrainian(t *testing.T) {
 	runTests(t, "uk", tests)
 }
 
+func TestCroatianBosnianSerbian(t *testing.T) {
```

Comment:
I would prefer to have 3 top level test functions, one for each language, so the number of tests matches the number of langugages. You can write a helper method that just does the same checks for each of the 3 tests.